### DISCUSSION TITLE
Tasks should be able to block merging if unchecked

### DISCUSSION THREAD

**Discussion by sferhan** [community/community#10224] [Pull Requests]
If there is an unchecked task in the PR description, it should block the PR merge. May not be the default behavior but should be configurable.

![image](https://user-images.githubusercontent.com/40257611/149775903-4713cb56-42f1-4f64-8fd9-9964493a9cc5.png)

**Comment by equalizer999** [comment]
Hi @sferhan, I just took your feedback and looped it back into our internal feedback system. Thanks for sharing!

**Reply by pieterdd** [reply]
Is there any news about this? We currently emulate this behavior with a lambda function that receives pull request edit events.
